ui/src/pages/ShoeMap.tsx

```diff
@@ -101,6 +102,10 @@ export const ShoeMap: React.FC = () => {
   const minZoom = 4;
   const maxZoom = 12;
   const currentLocation = { lat: 53.655697, lng: -100.13316 };
+  const southWest = L.latLng(40.712, -50.227),
+    northEast = L.latLng(70.774, -150.125),
+    mapBounds = L.latLngBounds(southWest, northEast);
```

Comment:
I would declaring them individually for +readability
```suggestion
  const southWest = L.latLng(40.712, -50.227);
  const northEast = L.latLng(70.774, -150.125);
  const mapBounds = L.latLngBounds(southWest, northEast);
```